jni/patches/cadical-shared.patch

```diff
@@ -13,7 +13,7 @@ index e761d26..43320d2 100755
 +	\$(MAKE) -C "\$(CADICALBUILD)" cadical.dll
 +.PHONY: all cadical clean mobical test update shared dll
  EOF
- 
```

Comment:
I do not think it is OK to remove this spaces. The process of patch application is very fragile, so it is better not to touch the patch files by hand.
Just do not commit changes on this patch file, since you are not doing any modifications to it at all.